### DISCUSSION TITLE
Mehrere Instanzen mit unterschiedlichen Versionen

### DISCUSSION THREAD

**Discussion by tschortschde** [Zefau/ioBroker.jarvis#2523] [Q&A] [answered]
Guten Morgen,
gibt es eine Möglichkeit bei Jarvis 2 Instanzen mit zwei Versionen zu installieren?
Wie ist das möglich?
VG

**Comment by Zefau** [comment, marked as answer]
Das erlaubt iobroker technisch nicht. Das geht mit keinem Adapter.
